ttb_resources/assets/ttb/lang/nl_nl.json

```diff
@@ -0,0 +1,91 @@
+  "item.ttb.mysterious_book.what_was_she_trying_to_build.page_0": "Wat probeerde zij te bouwen?",
+  "item.ttb.mysterious_book.where_am_i": "Waar ben ik?",
+  "item.ttb.mysterious_book.where_am_i.page_0": "De dagen zijn me voorbij gegaan En ik heb dagen verloren hierin Mary is niet meer",
+  "item.ttb.mysterious_book.where_am_i.page_1": "Ziekte verspreidde hij hier Bedorven door zwarte doodslag Als je het zo noemt \",\"Doei familie Iemand achtervolgt me Het is tijd om te gaan, liefde",
+  "item.ttb.mysterious_book.where_am_i.page_2": "Liefde",
+  "item.ttb.mysterious_book.where_am_i.page_4": ".....liefde.....",
+  "item.ttb.mysterious_crystal": "Mysterieuze Kristal",
+  "item.ttb.mysterious_dust": "Mysterieus Stof",
+  "item.ttb.optimism": "Optimisme",
+  "item.ttb.quick_boots": "Vlugge Laarzen",
+  "item.ttb.quick_bow": "Vlugge Boog",
+  "item.ttb.quick_sword": "Vlug Zwaard",
+  "item.ttb.restoration_tome": "Restauratie Boek",
+  "item.ttb.staff": "%s Staff",
+  "item.ttb.staff.blight": "Blight",
+  "item.ttb.staff.captive": "Gevangen",
+  "item.ttb.staff.ether": "Ether",
+  "item.ttb.staff.extraction": "Extractie",
+  "item.ttb.staff.harvest": "Oogst",
+  "item.ttb.staff.harvest.regress.took_lifeforce": "Niet genoeg ervaring om goed terug te gaan - je levenskracht heeft de kosten gecompenseerd",
+  "item.ttb.staff.harvest.regressed_spawn_egg.lore": "Teruggegaan van volwassenheid",
+  "item.ttb.staff.infallible": "Onfeilbaar",
+  "item.ttb.staff.mage": "Mage",
+  "item.ttb.staff.netherknight": "Netherridder",
+  "item.ttb.staff.refrigerant": "Afkoelende",
+  "item.ttb.staff.relativity": "Relativiteit",
+  "item.ttb.staff.repulsion": "Afstoting",
+  "item.ttb.staff.saturation": "Verzadiging",
+  "item.ttb.staff.shatter": "Breek",
+  "item.ttb.staff.skitter": "Skitter",
+  "item.ttb.staff.spirit": "Geest",
+  "item.ttb.staff.temporal": "Tijdelijke",
+  "item.ttb.staff.temporal.displaced_entity": "%s in tijd verplaatst.",
+  "item.ttb.staff.temporal.sped_up_time": "%s versnelde tijd",
+  "item.ttb.vulcaanium": "Vulcaanium",
+  "item.ttb.vulcaanium_crystal": "Vulcaanium Crystal",
+  "structure.ttb.deadlock_tower": "Deadlock Toren",
+  "structure.ttb.large_wooden_snow_dungeon": "Deadlock Kerker",
+  "structure.ttb.prismarine_stray_spawner": "Verdwaalde Spawner",
+  "structure.ttb.snow_blight": "Blight Boog",
+  "structure.ttb.snow_columns": "Sneeuw Kolommen",
+  "structure.ttb.snow_fort": "Sneeuw Fort",
+  "structure.ttb.snow_glacier": "Gletsjer",
+  "structure.ttb.snow_house": "Dorpelingenhut",
+  "structure.ttb.snow_igloo": "Sneeuw Kamp",
+  "structure.ttb.snow_spikes": "Sneeuw Kegel",
+  "structure.ttb.snowy_glass_lab": "Ondergronds Laboratorium",
+  "structure.ttb.stray_dungeon": "Verdwaalde Kerker",
+  "structure.ttb.stray_tower": "Verdwaalde Toren",
+  "text.ttb.debug.entity_removal": "Entity verwijderen: %s",
+  "text.ttb.entity.from": "%s van %s",
+  "text.ttb.entity.health_bar": "[%s] %s/%sgezondheidspunten",
```

Comment:
Is there a better way to write this? It shows up above an entity by default as, for example, `[Mage] 100/150HP`

Reply:
Imagine this with that translation lmao:
![image](https://user-images.githubusercontent.com/36608902/86020100-61748480-ba1f-11ea-8e74-f81f4ce2800c.png)

Reply:
let's just leave it as hp then, there is no well known-ish equivalent that is short